strangerstudios/pmpro-state-dropdowns
# Add United Kingdom (GB) states

### All Submissions:






### Changes proposed in this Pull Request:
Add United Kingdom (GB) counties to states array.



When "United Kingdom" is selected as a billing country the States field now provides a list of available states for that selection.

### How to test the changes in this Pull Request:

1. Navigate to PMPro Checkout.
2. Select "United Kingdom" from the Country select field.
3. See that the States field is now a select-type field with selectable values.

### Other information:




### Changelog entry

> Enhancement: Added states for "United Kingdom"
